### ISSUE TITLE
Bigger stickers preview

### ISSUE THREAD

**Issue by suraj3486** [rzhou1999/woosticker#5]
Please make preview of stickers bigger than current one. If a folder containing 99 stickers files than it should be preview as 33*3 in vertical scrolling. Gboard and most of the stickers app give vertical scrolling due to long phones.

Right now it is 3*33 in horizontal scrolling.

Some stickers contain quotes in it with bigger preview it will be easy in selecting stickers.

Thanks

**Comment by rzhou1999** [comment]
Good idea. It should be fairly easy to make this configurable by the user. Will look into when I have time....

Alternatively, if someone wants to submit a PR, I would suggest looking in ImageKeyboard.java and changing the for-loops to no longer be hardcoded to 3. Then, each image button layout needs to have its height/width scaled accordingly.

**Comment by rzhou1999** [comment]
See new release.

**Comment by suraj3486** [comment]
New release is quite good.
But there is no option for vertical scrolling or horizontal scrolling.
And horizontal scrolling is default🙄🙄

**Comment by rzhou1999** [comment]
I really don't like the idea of putting a vertical scroll in a keyboard that appears right below presumably top-to-bottom chat windows (why stack two vertical scrolls on top of each other?). It's definitely possible to add another toggle for this, but it's pretty low priority for me. If you feel so inclined feel free to send a PR but I probably won't implement it myself until after the UI gets an update.

**Comment by suraj3486** [comment]
Earlier even Gboard in Android was having horizontal scrolling everywhere which was quite sucking. But now they have made vertical scrolling mandatory every where. Which is quite easy to use

**Comment by rzhou1999** [comment]
That's a decision that I personally don't agree with so for now I think there's other areas that need attention first. I guess it is a useful configuration to have, but as I said before this is low priority for me so until I get a lot of free time. In the meantime, if anyone wants to look into it themselves, I believe you would need to look into changing the HorizontalScrollView in keyboard_layout.xml and correspondingly make row version of image_container_column.xml. Not sure how it would work with programmatically switching them in ImageKeyboard.java though.

**Comment by FredHappyface** [comment]
is this something you'd be happy for me to take a look at over the coming days?
